### PR TITLE
6 otp verification screen

### PR DESCRIPTION
**Pull request by ushamsharath**

### CODE REVIEW

File: lib/main.dart
Comment (maheshalyana) [inline lib/main.dart:1]: there should be now canges in main.dart

**Review by maheshalyana — CHANGES_REQUESTED** [review]

File: lib/main.dart
Comment (ushamsharath) [reply lib/main.dart:1]: updated the changes

**Review by maheshalyana — APPROVED** [review]

**Review by Abhi-0987 — COMMENTED** [review]
image

Font should be changed.. moving to development

**Review by maheshalyana — APPROVED** [review]

**Review by Abhi-0987 — COMMENTED** [review]
image
image

Design matched and test cases are passed.. moving to acceptance.

**Pull request merged**